keyboards/thumbsup/keymaps/default/keymap.c

```diff
@@ -0,0 +1,179 @@
+ * (Lower or Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |      |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  |BackSp|
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |   F1 |   F2 |   F3 |   F4 |   F5 |   F6 |   F7 |   F8 |   F9 |   F10| Enter|
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |  F11 |  F12 | LAlt |LShift| LAlt | RAlt |RShift|   ,  |   .  |   /  |      |
+ * `------+------+------+------+------+------+------+------+------+------+------+------.
+ *               |      |      |      |      |      |      |      |      |
+ *               `-------------------------------------------------------'
+ */
+[_NUMBERS] = LAYOUT(
+  _______, KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC,
+  LT(_EXTRARIGHT,KC_TAB), KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  _______,
+  _______, KC_F11,  KC_F12,  KC_LALT, KC_LSFT, KC_LALT, KC_RALT, KC_RSFT, KC_COMM, KC_DOT,  KC_SLSH, _______,
+	   _______, _______, _______, _______, _______, _______,  _______, _______, _______, _______
+),
+
+/* ExtraRightSide (the keys from the right side which did not fit into the matrix)
+ * ,-----------------------------------------------------------------------------------.
+ * | Esc  |BREAK |PLOVR2|PLOVER|TxBOLT|TxBlt2|   `  |   =  |   (  |   )  |   -  |      |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |******|      |      |      |      |      |PrtScr|      |      |      |  '   |      |
+   This is the
+   key turning
+   this layer on
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |      |      |NumPad|Qwerty|      |      |      |   [  |   ]  |  \   |      |
+ * `------+------+------+------+------+------+------+------+------+------+------+------.
+ *               |      |      |      |      |      |      |      |      |
+ *               `-------------------------------------------------------'
+ */
+[_EXTRARIGHT] = LAYOUT(
+  _______, KC_BRK,  XXXXXXX, XXXXXXX,      XXXXXXX, 	 XXXXXXX, 	KC_GRV,  KC_EQL,  KC_LPRN, KC_RPRN, KC_MINS, _______,
+  _______, XXXXXXX, XXXXXXX, XXXXXXX,      XXXXXXX,      XXXXXXX,       KC_PSCR, XXXXXXX, XXXXXXX, XXXXXXX, KC_QUOT, _______,
+  _______, XXXXXXX, XXXXXXX, XXXXXXX,      XXXXXXX,      XXXXXXX,       XXXXXXX, XXXXXXX, KC_LBRC, KC_RBRC, KC_BSLS, _______,
+	   _______, _______, _______,      _______,      _______,       _______,  _______, _______, _______, _______
+)
+};
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+  return update_tri_layer_state(state, _CURSORLEFT, _CURSORRGHT, _NUMBERS);
+}
+                                                                            
```

Comment:
The tri-layer feature should be used instead. If you want tri-layer functionality, it needs to be enabled in a keymap level rules.mk file using `TRI_LAYER_ENABLE = yes` and utilised with the `TL_LOWR`/`TL_UPPR` keys.
```suggestion
```

Reply:
Thank you, LT(TL_LOWR,KC_SPC) works.

(I used that layer_state_set_user() call on Drashna's reply to my question post on Reddit, so I thought it was kosher. Anyway, since LT() works it is good for me.)